MAUI/NavigationDrawer/Side-Pane-Sizing.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: post
+title: Setting Sliding Panel size in .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about Setting Sliding Panel size support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Setting Sliding Panel size in .NET MAUI Navigation Drawer
+
+The size of the side pane can be adjusted using the [`DrawerHeight`]() and [`DrawerWidth`]() properties.
+
+## Drawer Height
+
+The [`DrawerHeight`]() property is used to change the height of the side pane when the [`Position`]() is set to [`Top`]() or [`Bottom`]().
```

Comment:
The `DrawerHeight` property is used to adjust the height of the side pane when the `Position` is set to either `Top` or `Bottom.`